drivers/kw2xrf/Makefile.dep

```diff
@@ -1,7 +1,8 @@
-USEMODULE += luid
+USEMODULE += core_thread_flags
 USEMODULE += ieee802154
+USEMODULE += luid
 USEMODULE += netdev_ieee802154
-USEMODULE += core_thread_flags
+USEMODULE += netdev_legacy_api
```

Comment:
This now uses the radio HAL, so no need to select this here anymore

Reply:
I rebased now to have this on top of the conversion to the radio HAL. I could have just dropped this change and this would have avoided the merge conflict, but with rebasing `git bisect` should be more fun :)